s3/pkg/datastore/azure/azure.go

```diff
@@ -142,10 +185,10 @@ func (ad *AzureAdapter) Put(ctx context.Context, stream io.Reader, object *pb.Ob
 }
 
 func (ad *AzureAdapter) Get(ctx context.Context, object *pb.Object, start int64, end int64) (io.ReadCloser, error) {
-	bucket := ad.backend.BucketName
+	bucket := object.BucketName
 	log.Infof("get object[Azure Blob], bucket:%s, objectId:%s\n", bucket, object.ObjectId)
-
-	blobURL := ad.containerURL.NewBlobURL(object.ObjectId)
+	containerURL, _ := ad.createBucketContainerURL(ad.backend.Access, ad.backend.Security, object.BucketName)
```

Comment:
why do we need createBucketContainerURL instead of createContainerURL?

Reply:
because createContainerURL doesnt involve cloud bucket name in URL
but for creating the cloud bucket,container URL should  contain bucket name in it

Reply:
ok